webui/src/App.js

```diff
@@ -18,8 +22,11 @@ function App() {
 
   return initialized ? (
     <BrowserRouter>
+    <NavBar />
       <Routes>
         <Route path="/" element={<Home />} />
```

Comment:
Can you try something like this?
`<Route path="/" element={<Navigate to="/pools" replace />} />` This should redirect the user to pools page.

Reply:
Ok, I will try that out.